test/configCases/css/minimize-cssnano-custom-properties/style.css

```
@@ -0,0 +1,42 @@
+/*! colormin: a substitution names no color to shorten */
+h1 { color: var(--white) }
+/*! colormin: a custom property's channels are not a color function */
+:root { --some-color: 200 255 0 }
+/*! colormin: a color function in a custom property (cssnano rewrites, we ask) */
+a { --foo: rgb(0 0 0) }
+/*! colormin: an alpha carried by a substitution keeps the function */
+div { background-color: rgba(51,153,255,var(--tw-bg-opacity)) }
+/*! whitespace: a nested parenthesis inside calc() */
+.w1 { width: calc(10px - ( 100px / var(--test) )) }
+/*! whitespace: an upper-case CALC( is still calc() */
+.w2 { width: CALC(10px - ( 100px / var(--test) )) }
```

Stylelint (17.14.0)

[error] Expected "CALC" to be "calc" (function-name-case)

(function-name-case)